### DISCUSSION TITLE
7 Day Power State color change

### DISCUSSION THREAD

**Discussion by acrey1** [Ylianst/MeshCentral#6252] [Ideas]
Hi Meshers,

I would like to change the colors for the 7 Day Power State to be a bit more intuitive (e.g. Green [0,128,0] = power ON, Red [255,0,0] = power OFF, Yellow [255,255,0] =  SLEEP etc).

Any ideas how this can be done?

**Comment by si458** [comment]
Sadly it's hard coded into the default.handlebars file
BUT open a githuh issue as a enhancement request and we can look at moving the hard coded colors into the css file. Then u can customise the css file to ur hearts content with colours hehe

**Comment by si458** [comment]
the css is already in there, follow steps here to change them
https://github.com/Ylianst/MeshCentral/issues/6253#issuecomment-2225239532
